### ISSUE TITLE
Generated enums are not brought into scope in prisma.rs

### ISSUE THREAD

**Issue by westernwontons** [Brendonovich/prisma-client-rust#212]
Generating the client for MongoDB doesn't bring enums into scope. I can manually import them and the client seems to be working fine (haven't fully tested tho).

For example:
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/75550932/203851399-5f29993e-9138-4a29-baf8-9d00290d367b.png">

Is this a bug?

**Comment by alakhpc** [comment]
running into the same issue with postgresql

reproduction:
```prisma
datasource db {
    provider = "postgresql"
    url      = env("PRISMA_DATABASE_URL")
}

generator rust {
    provider = "cargo prisma"
    output   = "./src/prisma.rs"
}

enum ProductType {
    New
}

model Test {
    id           Int         @id
    product_type ProductType
}
```

```
error[E0412]: cannot find type `ProductType` in this scope
   --> src/prisma.rs:653:23
    |
653 |             InVec(Vec<ProductType>),
    |                       ^^^^^^^^^^^ not found in this scope
    |
help: consider importing this enum
    |
596 |         use crate::prisma::ProductType;
    |
```

**Comment by jqphu** [comment]
I'm also running into this using postgresql. Currently just working around it by manually bringing in the enum into scope.

**Comment by Brendonovich** [comment]
Duplicate of #197. Has been fixed on 1.6.4 branch.

**Comment by jqphu** [comment]
Awesome, just switched to using 0.6.4 branch and it works perfectly. Thanks!
